### ISSUE TITLE
2023-03-08: English/Deep learning terms

### ISSUE THREAD

**Issue by dsmits** [carpentries/glosario#576]
I am currently working on the glossary for the [deep learning lesson](https://github.com/carpentries-incubator/deep-learning-intro).

There are a couple of terms that I need that are currently missing from glosario:

- artificial intelligence
- convolutional neural network (CNN)
- recurrent neural network (RNN)
- epoch
- learning rate
- class imbalance
- hidden layers

**Comment by dsmits** [comment]
I can write these definitions myself, just let me know if it's ok if I add these.

**Comment by elletjies** [comment]
@dsmits please go ahead and submit the terms :)

**Comment by svenvanderburg** [comment]
@elletjies #584 is merged, but I don't see the new terms appearing in glosario. For example: https://glosario.carpentries.org/en/#cnn

I see in the Github Action that Deploy was skipped: https://github.com/carpentries/glosario/actions/runs/5875720776/job/16019059004 

How do we get this to appear in the glosario?

**Comment by elletjies** [comment]
@svenvanderburg, it seems to appear in the glossary.yml file but not on https://glosario.carpentries.org. @zkamvar will you be able to assist?

**Comment by zkamvar** [comment]
I've opened https://github.com/carpentries/glosario/pull/596 to fix the issue and I've tested it on my own fork and confirm that it works.

**Comment by elletjies** [comment]
Thank you for fixing this @zkamvar!

**Comment by svenvanderburg** [comment]
All the terms added in this issue still do not show up, for example this term is still not in there: https://glosario.carpentries.org/en/#cnn Could you reopen this issue? @elletjies @zkamvar

**Comment by svenvanderburg** [comment]
It seems like the yaml was not formatted correctly, I think this fixes it: https://github.com/carpentries/glosario/pull/598

**Comment by svenvanderburg** [comment]
Yes, it is fixed now! Thanks a lot @elletjies and @zkamvar :pray:

**Comment by elletjies** [comment]
@all-contributors please add @dsmits  question

**Comment by elletjies** [comment]
@all-contributors please add @svenvanderburg  question
